ui/admin/tests/helpers/selectors.js

```diff
@@ -15,6 +15,8 @@ export const FIELD_DESCRIPTION = '[name=description]';
 export const FIELD_DESCRIPTION_VALUE = 'description';
 
 export const TABLE_RESOURCE_LINK = (url) => `tbody [href="${url}"]`;
+export const TABLE_FIRST_ROW_ACTION_DROPDOWN =
+  'tbody tr:first-child td:last-child button';
```

Comment:
I saw this selector in another resource specific file. I plan to have a PR to move it just here since this is a table specific selector but not unique to any resource.